p/scripts/main.js

```diff
@@ -447,9 +447,7 @@ function toggleContent(new_active, old_active, skipping) {
 			// when skipping, this feels more natural if it’s not so near the top
 			new_pos -= document.body.clientHeight / 4;
 		}
-		if (relative_move) {
```

Comment:
Isn't that used in normal view?

Reply:
see line 425: `const relative_move = context.current_view === 'global';`